### DISCUSSION TITLE
Current sort column

### DISCUSSION THREAD

**Discussion by jclijmans** [quasarframework/quasar#13234] [General - Components / Directives / etc] [answered]
I've got a table with client-side sorting.
I want to persist the sort column so when returning to the page, I can restore the sorting.

How can I get the current sort column and direction?

As far as I can see, there is no sort event, nor a sort column property.

**Comment by metalsadman** [comment, marked as answer]
save it in memory using pinia or in local storage.

**Reply by jclijmans** [reply]
Sorry that I didn't express my self clearly enough.

It's not that I can't persist data, I already use Pinia for that.
I just don't know where to get the column name and direction by which the table is sorted.

As far as I know, there is no property, nor event to give me back the current sort column or name.
There is the "sort-method", but that means I have to sort myself.

I would like to know where I can find the column name and direction the table is currently sorted with so I can save that in the pinia store and restore it when returning to the page.

Thanks

**Reply by edgarasne** [reply]
Try v-model:pagination which has sortKey pair it with watch and that's it.
From docs: "pagination : Object
Description
Pagination object; You can also use the 'v-model:pagination' for synching; When not synching it simply initializes the pagination on first render"

Also you can always use refs

**Reply by jclijmans** [reply]
It works, thanks!!!!

I already tried to bind the pagination, but that didn't work.
Using v-model:pagination works like a charm, I'm now able to persist the sort column and direction.

Thanks!!
